unit_tests/test_data/test_scylla_yaml_builders/PR-provision-test.result.json

```diff
@@ -20,5 +20,6 @@
   "alternator_enforce_authorization": false,
   "auto_bootstrap": true,
   "enable_ipv6_dns_lookup": false,
+  "listen_interface": "eth0",
```

Comment:
i think on GCE network interfaces have different names

Reply:
we don't support multiple networks for GCP, so I don't expect it to be relevant.

I'll trigger those provision test to be on the safe side

Reply:
GCP passed nicely, 
Azure failed in some weird way and missing some part of the SCT logs, I'll trigger it again

Reply:
Also azure passed nicely, merging this one